components/form/GenerateFormItem.razor.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Rendering;
+using AntDesign.Internal;
+
+namespace AntDesign;
+
+public partial class GenerateFormItem<TModel> : ComponentBase
+{
+    private PropertyInfo[] _propertyInfos = { };
+
+
+    [CascadingParameter(Name = "Form")]
+    private IForm Form { get; set; }
+
+
+    /// <summary>
+    /// If true then the rules will be required by CheckTypeRequire.
+    /// Recommended only for forms without complex validation, where only simple validation for empty values is required.
+    /// </summary>
+    [Parameter]
+    public bool RulesRequireByType { get; set; }
```

Comment:
I think we don't need this parameter and use `ValidateMode` instead to set the validation.

Reply:
I agree with your perspective. We can remove this parameter and use ValidateMode instead to set the validation. I’ll make the necessary adjustments to the code, or have you already started making the changes?

Reply:
Not yet, please.